src/RoadRegistry.BackOffice.Api/Infrastructure/Configuration/FeatureToggleOptions.cs

```diff
@@ -4,5 +4,6 @@ public sealed class FeatureToggleOptions
     {
         public const string ConfigurationKey = "FeatureToggles";
         public bool UseFeatureCompare { get; set; }
+        public bool UseApiKeyAuthentication { get; set; }
```

Comment:
this is the one feature that doesn't need a feature flag :-)

Reply:
Indeed, however for local development, we can shut it down in appsettings.development.json. We can remove if need be?